larq/quantizers.py

```diff
@@ -72,8 +123,8 @@ def ste_sign(x):
     (essentially the binarization is replaced by a clipped identity on the
     backward pass).
     \\[\frac{\partial q(x)}{\partial x} = \begin{cases}
-      1 & \left|x\right| \leq 1 \\\
-      0 & \left|x\right| > 1
+      1 & \left|x\right| \leq t_\text{clip} \\\
```

Comment:
Should be clip_value?

Reply:
I used clip_value here before and it looked not very good in the rendered version. I could live with a monospaced version that shows that it relates to the variable used in the code:
<img width="688" alt="Bildschirmfoto 2019-08-14 um 14 16 44" src="https://user-images.githubusercontent.com/13285808/63020405-56b01000-be9e-11e9-998c-1ff6332f8782.png">
I still prefer the t_clip since it is documented in the line below, what do you think?

Reply:
I would always rather be explicit and use clip_value to reduce confusion even though aesthetically it looks worse